File: tests/openmp_examples/parallel_execution/host_teams.1.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp teams num_teams(nteams_required) thread_limit(max_thrds) private(tm_id)
#pragma omp parallel
#pragma omp for
#pragma omp for simd simdlen(8)
#pragma omp parallel
#pragma omp for
#pragma omp for simd simdlen(4)

File: tests/openmp_examples/parallel_execution/host_teams.1.f90
!$omp    teams num_teams(nteams_required) thread_limit(max_thrds) private(tm_id)
!$omp          parallel
!$omp             do
!$omp             do simd simdlen(8)
!$omp          end parallel
!$omp          parallel
!$omp             do
!$omp             do simd simdlen(4)
!$omp          end parallel
!$omp    end teams
